packages/flame/test/text_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flame/components.dart';
+import 'package:test/test.dart';
+
```

Comment:
We should add a small text to check if the default registration is working properly.